data/pref.xml

```diff
@@ -46,6 +46,10 @@
       <value id="ACTIVE_LAYER" value="0" />
       <value id="ALL_LAYERS" value="1" />
     </enum>
+    <enum id="EightConnected">
```

Comment:
You could rename this enum to `PixelConnectivity` 👍 
```suggestion
    <enum id="PixelConnectivity">
```
So the pixel connectivity can be 8-connected or 4-connected.